### PR TITLE
docs: improve documentation - bug, typo, correctness

### PR DESCRIPTION
**Pull request by JessicaXYWang**
## Related Issues/PRs

# Fix typo, correctness and bug for website and preparing notebooks for Azure doc 

## What changes are proposed in this pull request?

_Briefly describe the changes included in this Pull Request._

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

1. Find the corresponding markdown file for your new feature in `website/docs/documentation` folder.
   Make sure you choose the correct class `estimators/transformers` and namespace.
2. Follow the pattern in markdown file and add another section for your new API, including pyspark, scala (and .NET potentially) samples.
3. Make sure the `DocTable` points to correct API link.
4. Navigate to website folder, and run `yarn run start` to make sure the website renders correctly.
5. Don't forget to add `<!--pytest-codeblocks:cont-->` before each python code blocks to enable auto-tests for python samples.
6. Make sure the `WebsiteSamplesTests` job pass in the pipeline.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/mlflow/examples.md](https://github.com/microsoft/SynapseML/blob/3b2847a1827bdb82d4c6761d6b465c72810d17d4/website%2Fdocs%2Fmlflow%2Fexamples.md) | 65 | 100 | 51 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2fb7a153-7bc1-4856-b97f-c1ba7eaa1351) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by JessicaXYWang** [comment]
no code change, all documentation. just FYI.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/mlflow/examples.md](https://github.com/microsoft/SynapseML/blob/39f34e3bbdbc2a3c5a3dc4d18d0971c0e9bb1f21/website%2Fdocs%2Fmlflow%2Fexamples.md) | 65 | 100 | 51 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bb784dc8-e9c5-4c8c-8bbb-cf3e8f74d1e9) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: notebooks/features/classification/Classification - Sentiment Analysis Quickstart.ipynb [hunk 1]
```diff
@@ -1,287 +1,296 @@
...
-       "execution_finish_time": "2022-09-02T20:18:21.4549018Z",
-       "spark_jobs": null
+       "spark_jobs": null,
+       "spark_pool": null,
+       "state": "cancelled",
+       "statement_id": null
       },
-      "text/plain": "StatementMeta(, , , Cancelled, )"
+      "text/plain": [
+       "StatementMeta(, , , Cancelled, )"
```
Comment (mhamilton723) [inline notebooks/features/classification/Classification - Sentiment Analysis Quickstart.ipynb:81]: in a future PR we should remove these weird output cells that clutter the doc

**Review by mhamilton723 — APPROVED** [review]
wooowwww

**Review by serena-ruan — APPROVED** [review]
Looks great! Thanks!

**Review by eisber — APPROVED** [review]
thanks!

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/mlflow/examples.md](https://github.com/microsoft/SynapseML/blob/7d587584b408a46b23fe094c279ebfb7c395deb7/website%2Fdocs%2Fmlflow%2Fexamples.md) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d18ab89-a49c-4c43-8271-889a91c9c1fc) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**